### PR TITLE
Regenerate turtle graphics panel when walk namespace is reloaded, if needed

### PR DESCRIPTION
**Pull request by Engelberg**

### CODE REVIEW

File: src/clojurebridge_turtle/core.clj [hunk 1]
```diff
@@ -43,11 +43,12 @@
...
-                                 :y 0
-                                 :angle 90
-                                 :color color}))
+         (dosync
+          (swap! lines assoc n [])
+          (swap! turtles assoc n {:x 0
+                                  :y 0
+                                  :angle 90
+                                  :color color})))
        {n (n @turtles)})))
```
Comment (yokolet) [inline src/clojurebridge_turtle/core.clj:51]: @brandonbloom deleted `dosync` at the pull request https://github.com/ClojureBridge/welcometoclojurebridge/pull/3 . `dosync` does some trick?

**Comment by yokolet** [comment]
Thanks @Engelberg ! I believe you checked turtles' app works well with this fix. But, let me try before click on pull request.

File: src/clojurebridge_turtle/core.clj (hunk 1 above)
Comment (yokolet) [inline src/clojurebridge_turtle/core.clj:51]: OK. got it. Github says "commented on an outdated diff"

Comment (Engelberg) [inline src/clojurebridge_turtle/core.clj:51]: Nope, dosync is not needed here.  I was working from a file that predated brandonbloom's changes.  But when I generated the pull request, I saw he had taken that out, and so I added further commits to the pull request to mimic what Brandon did.  So the pull request includes one commit that appears to put the dosync back in, followed by one that takes it out -- no net change in the overall diff.

**Comment by yokolet** [comment]
Looks good. Sometimes, `undo` doesn't work well, but I don't think this pr is the reason. I'm going to merge this pull request.

**Pull request merged**